backend/app/__init__.py

```
@@ -0,0 +1,16 @@
+from flask import Flask
+from flask_jwt_extended import JWTManager
+from .api_routes.auth import auth_bp
+import os
+from dotenv import load_dotenv
+
+def create_app():
+    app = Flask(__name__)
+    app.config['SECRET_KEY'] = os.getenv("SECRET_KEY")
+    app.config['JWT_SECRET_KEY'] = os.getenv("JWT_SECRET_KEY")
```

Add validation for required environment variables.

The application will fail silently if required environment variables are not set. Add validation to ensure proper configuration.